Dockerfile

```diff
@@ -54,6 +54,20 @@ RUN \
   make && \
   ln -s `pwd`/dirb /usr/local/bin/dirb
 
+#Install gobuster
+RUN \
+ cd / && \
+ wget -q https://github.com/OJ/gobuster/releases/download/v3.0.1/gobuster-linux-amd64.7z && \
+ apt update && \
+ apt install -y p7zip-full && \
```

Comment:
I am updating apt not apt-get.
I then use apt to install p7zip, it will not install using apt-get.

I will omve the update and install commands to a more appropriate place.

Reply:
I have made the changes and tested the image. Please merge